packages/components/src/components/select/select.ts

```diff
@@ -1041,7 +1041,7 @@ export default class SdSelect extends SolidElement implements SolidFormControl {
       }
 
       sd-popup::part(popup) {
-        @apply overflow-y-scroll;
+        @apply overflow-y-scroll z-dropdown;
```

Comment:
Just wondering what z-dropdown does, due to the fact that I'm not able to find any documentation about it 🤔

Reply:
You can find it in **_create-theme.cjs_**:
<img width="437" alt="Screenshot 2024-01-17 at 14 24 01" src="https://github.com/solid-design-system/solid/assets/20210155/c89d7aa7-e289-4c13-8988-e081cb61b49c">

Reply:
thx